packages/hooks/src/useCurrencyConfig.ts

```diff
@@ -77,7 +82,6 @@ const useCurrencyConfig = () => {
         getConfig,
         /** Available currencies and their information */
         currencies_config,
-        ...rest,
```

Comment:
@sergei-deriv Why did you delete this?

Reply:
@mahdiyeh-deriv it's because now we have 2 requests instead on one and I don't know which one I need to choose :) but ok, I'll back it for 2nd one, for crypto_config